src/Std/Data/HashMap/Lemmas.lean

```diff
@@ -698,6 +702,51 @@ theorem distinct_keys [EquivBEq α] [LawfulHashable α] :
     m.keys.Pairwise (fun a b => (a == b) = false) :=
   DHashMap.distinct_keys
 
+@[simp]
+theorem length_toList [EquivBEq α] [LawfulHashable α] :
+    m.toList.length = m.size :=
+  DHashMap.Const.length_toList
+
+@[simp]
+theorem isEmpty_toList [EquivBEq α] [LawfulHashable α] :
+    m.toList.isEmpty = m.isEmpty :=
+  DHashMap.Const.isEmpty_toList
+
+theorem mem_toList_iff_get?_eq_some [LawfulBEq α]
+    {k : α} {v : β} :
+    (k, v) ∈ m.toList ↔  m[k]? = some v :=
```

Comment:
```suggestion
    (k, v) ∈ m.toList ↔ m[k]? = some v :=
```

Reply:
Also, I thought that for now we were still naming such lemmas using `getElem?` rather than `get?`. I'm not opposed to fast-forwarding, as this is the name we eventually want to have.